package.json

```diff
@@ -7,9 +7,9 @@
   },
   "scripts": {
     "cleanup": "del-cli db.json",
-    "start": "hexo generate --config docs_config.yml --watch & node server.js & wait",
+    "start": "hexo clean && hexo generate --config docs_config.yml,blog_config.yml --watch & node server.js & wait",
```

Comment:
Should `cleanup` be changed to `hexo clean` and then we can add it to `prestart` like `prebuild`?